core/src/i18n/en.yaml

```diff
@@ -1,5 +1,6 @@
 addons:
   addons-status: Addons Status
+  description: <0>Addons Configuration</0> defines Namespace-scoped addons that provide all information required to convert Helm charts into Service Classes in Service Catalog.
```

Comment:
`Namespace-scoped addons` I'm not a fan of the capitalized hyphenated adjective, but I guess character constraints made that necessary?

Reply:
We also capitalize the term "Namespace" in our docs, since it's a Kubernetes term. See the [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#capitalized-terms) for reference.

Reply:
I know :sob: it just looks less weird if it's a noun, not part of an adjective phrase. But I'm not going to object :D